src/validation.cpp

```diff
@@ -1118,7 +1118,8 @@ void UpdateCoins(const CTransaction& tx, CCoinsViewCache& inputs, CTxUndo &txund
         txundo.vprevout.reserve(tx.vin.size());
         BOOST_FOREACH(const CTxIn &txin, tx.vin) {
             txundo.vprevout.emplace_back();
-            inputs.SpendCoin(txin.prevout, &txundo.vprevout.back());
+            bool fSpent = inputs.SpendCoin(txin.prevout, &txundo.vprevout.back());
```

Comment:
In commit "Return a bool in SpendCoin to restore pre-per-utxo assert semantics"

Maybe drop hungarian.